doc/tutorials/notifications.md

```diff
@@ -257,6 +257,116 @@ int main (void)
 }
 ```
 
+### How-To: Reload KDB when Elektra's configuration has changed
+
+This section shows how the notification feature can be used to reload an
+application's KDB instance when Elektra's configuration has changed.
+This enables applications to apply changes to mount points or globally mounted
+plugins without restarting.
+
+**Step 1: Register for changes to Elektra's configuration**
+
+To achieve reloading on Elektra configuration changes we register for changes
+below the key `system/elektra` using
+`elektraNotificationRegisterCallbackSameOrBelow()`.
+
+```C
+Key * elektraKey = keyNew ("system/elektra", KEY_END);
+elektraNotificationRegisterCallbackSameOrBelow (kdb, elektraKey, elektraChangedCallback, NULL))
+keyDel (elektraKey);
+```
+
+**Step 2: Create a function for reloading KDB**
+
+Since our application needs to repeatedly initialize KDB on
+configuration changes we need to create a function which cleans
+up and reinitializates KDB.
+
+```C
+void initKdb (void)
+{
+	if (kdb != NULL)
+	{
+		// Cleanup notifications and close KDB
+		elektraNotificationClose (kdb);
+		kdbClose (kdb, parentKey);
+	}
+
+	kdb = kdbOpen (parentKey);
+	elektraIoSetBinding (kdb, binding);
+	elektraNotificationOpen (kdb);
+
+  // registration code from snippet before
+	Key * elektraKey = keyNew ("system/elektra", KEY_END);
+	elektraNotificationRegisterCallbackSameOrBelow (kdb, elektraKey, elektraChangedCallback, NULL);
+	keyDel (elektraKey);
+
+  // TODO: add application specific registrations
+
+	// Get configuration
+	kdbGet (kdb, config, parentKey);
+}
+```
+
+**Step 3: Handle configuration changes**
+
+The last step is to connect the registration for changes to the
+(re-)initialization function.
+Directly calling the function is discouraged due to tight coupling (see
+guidelines in the next section) and also results in an application crash since
+the notification API is closed while processing notification callbacks.
+Therefore we suggest to add a timer operation with a sufficiently small interval
+which is enabled only on configuration changes.
+This timer will then call the initialization function.
+
+First, we create the timer in the main loop setup of the application.
+
+```C
+// (global declaration)
+ElektraIoTimerOperation * reload;
+
+// main loop setup (e.g. main())
+reload = elektraIoNewTimerOperation (100, 0, initKdb, NULL);
```

Comment:
Why 100?

Reply:
The timer interval is 100 milliseconds; I'll add a comment.